crates/cargo-contract/src/cmd/info.rs

```diff
@@ -74,6 +75,9 @@ pub struct InfoCommand {
         default_value = "ws://localhost:9944"
     )]
     url: url::Url,
+    /// A name of a production chain to upload or instantiate the contract on.
+    #[clap(name = "chain", long, conflicts_with = "url")]
```

Comment:
Apart from the URL, the chain could also retain the chain configuration. So, when you specify 'alephzero', 'moonbeam', or whatever, it would also set the correct `config` (https://github.com/paritytech/cargo-contract/blob/master/crates/cargo-contract/src/cmd/config.rs) for the chain.

In this case chain would conflict with `url` and `config`

Reply:
This can be a follow up PR

Reply:
Ok